CHANGELOG.rst

```diff
@@ -1,6 +1,60 @@
+- [API] Tags can now be set via a SetTag API, and they have been moved to ``RunData`` from ``RunInfo`` (#342, @aarondav)
+- [API] Added ``list_artifacts`` and ``download_artifacts`` to ``MlflowService`` to interact with a run's artifactory (#350, @andrewmchen)
+- [API] Added ``get_experiment_by_name`` to Python Tracking API, and equivalent to Java API (#373, @vfdev-5)
+- [API/Python] Version is now exposed via ``mlflow.__version__``.
+- [API/CLI] Added ``mlflow artifacts`` CLI to list, download, and upload to run artifact repositories (#391, @aarondav)
+- [UI] Added icons to source names in MLflow Experiments UI (#381, @andrewmchen)
+- [UI] Added support to view ``.log`` and ``.tsv`` files from MLflow artifacts UI (#393, @Shenggan; #433, @whiletruelearn)
+- [UI] Run names can now be edited from within the MLflow UI (#382, @smurching)
+- [Serving] Added ``-host`` option to ``mlflow serve`` to allow listening on non-local addressess (#401, @hamroune)
+- [Serving/SageMaker] Elastic Container Registry boto client can now take an AWS region (#366, @dbczumar)
+- [Python] Added environment variables to support providing HTTP auth (username, password, token) when talking to a remote MLflow tracking server (#402, @aarondav)
+- [Python] Added support to override S3 endpoint for S3 artifactory (#451, @hamroune)
+- MLflow nightly Python wheel and JAR snapshots are now available and linked from https://github.com/mlflow/mlflow (#352, @aarondav)
+
+Bug fixes and documentation updates:
+
+- [Python] ``mlflow run`` now logs default parameters, in addition to explicitly provided ones (#392, @mparkhe)
+- [Python] ``log_artifact`` in FileStore now requires a relative path as the artifact path (#439, @mparkhe)
+- [Python] Fixed string representation of Python entities, so they now display both their type and serialized fields (#371, @smurching)
+- [UI] Entry point name is now shown in MLflow UI (#345, @aarondav)
+- [Models] Keras model export now includes Tensorflow graph explicitly to ensure the model can always be loaded at deployment time (#440, @tomasatdatabricks)
+- [Python] Fixed issue where FileStore ignored provided Run Name (#358, @adrian555)
+- [Python] Fixed an issue where any ``mlflow run`` failing printed an extraneous exception (#365, @smurching)
+- [Python] uuid dependency removed (#351, @antonpaquin)
+- [Python] Fixed issues with remote execution on Databricks (#357, #361, @smurching; #383, #387, @aarondav)
+- [Docs] Added `comprehensive example <https://github.com/mlflow/mlflow/tree/master/examples/multistep_workflow>`_ of doing a multistep workflow, chaining MLflow runs together and reusing results (#338, @aarondav)
+- [Docs] Added code examples to ``mlflow.keras`` API (#341, @dmatrix)
+- [Docs] Significant improvements to Python API documentation (#454, @stbof)
+- [Docs] Examples folder refactored to improve readability. The examples now reside in ``examples/`` instead of ``example/``, too (#399, @mparkhe)
```

Comment:
Might be good to mention this separately, or in the new features; this is a pretty visible change